src/ui/pages/Menu/Menu.tsx

```diff
@@ -105,6 +108,10 @@ const Menu = () => {
     dispatch(setCurrentRoute({ path: TabsRoutePath.MENU }));
   });
 
+  useEffect(() => {
```

Comment:
This code is duplicate, Can we write it as new hook?
Or can we move it to `TabLayout` component ?